ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestApplication.java

```diff
@@ -391,6 +404,13 @@ private KsqlFailedPrecondition(final String message) {
     }
   }
 
+  static final class AbortApplicationStartException extends Exception {
```

Comment:
Why are we using a checked exception here?  We don't really use checked exceptions any more. They're a PITA! Why not extend `KsqlServerException`?